src/main/resources/add-ons-to-index.json

```diff
@@ -2987,6 +3017,7 @@
       "tags": [
         "messaging"
       ],
+      "icon": "comment-o",
```

Comment:
I would probably replace this with bell-o as it fits better(check out the wiki of the module for more insight)